src/class/ProgressTimer.js

```diff
@@ -0,0 +1,115 @@
+// @ts-check
+import generators from '@endpass/utils/generators';
+
+const DEFAULT_TOTAL_TIME = 20 * 1000; // 20 sec
```

Comment:
if we have dirty logic in the class, maybe should be placed near the controllers?
comment under the string